src/number_theory/arithmetic_function.lean

```diff
@@ -740,6 +829,35 @@ end
 
 end comm_ring
 
+/-- One version of Möbius inversion. -/
```

Comment:
From this name, you can guess that there is also a version without a prime `'`. But the converse is not true. So please add a cross-refs in the docstrings (and if possible briefly sketch how the statements differ).

Reply:
I've renamed them a bit to be a bit more descriptive, and put everything in the module doc.